dev-docs/cellxgene_census_draft_schema.md

```diff
@@ -901,7 +1012,7 @@ For each organism the `SOMAExperiment` MUST contain the following:
 * Obs to spatial data mapping:
   * Obs to spatial data – `census_obj["census_spatial_sequencing"][organism].obs_spatial_presence` – `SOMADataFrame`
 * Spatial data  –  `census_obj["census_spatial_sequencing"][organism].spatial` – `SOMACollection`.
-  * Spatial Scenes with spatial data  –  `census_obj["census_spatial_sequencing"][organism].spatial[scene_id]`  – `SOMAScene`.  There will be as many as Spatial Scenes as  spatial datasets. Each`SOMAScene` MUST contain the following:
+  * Spatial Scenes with spatial data  –  `census_obj["census_spatial_sequencing"][organism].spatial[scene_id]`  – `SOMAScene`.  There will be as many Spatial Scenes as  spatial datasets. Each`SOMAScene` MUST contain the following:
     * Positions array – `census_obj["census_spatial_sequencing"][organism].spatial[scene_id].obsl["loc"]` – `SOMAPointCloudDataFrame`.
     * High resolution image  – `census_obj["census_spatial_sequencing"][organism].spatial[scene_id].img[library_id]["highres_image"]` – `MultiscaleImage`.
```

Comment:
Might be a misnomer to call the `hires image` a high resolution image. It is actually a downsampled image that is called hires, only because it is higher than the lowres image. It is the full res image that is the highest resolution: https://www.10xgenomics.com/support/software/space-ranger/latest/analysis/outputs/spatial-outputs#tissue-png

Also, this section, in general, makes it sound like there should be images for Slide-seq. The CxG schema does not have images for Slide-seq.

Reply:
This is outside the scope of this PR which is focused on organism. I made a small editorial change that I noticed. I can open another issue for later spatial editorial work.